File: src/main/java/com/kenvix/utils/timer/Stopwatch.java
//--------------------------------------------------
// Class Stopwatch
//--------------------------------------------------
// Written by Kenvix <i@com.kenvix.com>
//--------------------------------------------------

package com.kenvix.utils.timer;

public class Stopwatch {
    private long beginTime;

    public long start() {
        return beginTime = System.currentTimeMillis();
    }

    public long getTimeDifference() {
        return System.currentTimeMillis() - beginTime;
    }
}

File: src/main/java/com/kenvix/utils/timer/Stopwatch.kt
//--------------------------------------------------
// Class Stopwatch
//--------------------------------------------------
// Written by Kenvix <i@com.kenvix.com>
//--------------------------------------------------
package com.kenvix.utils.timer

class Stopwatch {
    private var beginTime: Long = 0

    fun start(): Long {
        return System.currentTimeMillis().also { beginTime = it }
    }

    val timeDifference: Long
        get() = System.currentTimeMillis() - beginTime
}